app/components/search/SearchResults/SearchResults.tsx

```diff
@@ -2,52 +2,38 @@ import React, { useState, useEffect } from "react";
 import { Link } from "react-router-dom";
 import ReactMarkdown from "react-markdown";
 import { connectStateResults } from "react-instantsearch/connectors";
+import { Props as GoogleMapProps, Coords } from "google-map-react";
 import { whiteLabel } from "utils";
 import { SearchMap } from "components/search/SearchMap/SearchMap";
 import ResultsPagination from "components/search/Pagination/ResultsPagination";
 import { Texting } from "components/Texting";
 import { ClinicianActions } from "components/ucsf/ClinicianActions/ClinicianActions";
 import { ClientHandouts } from "components/ui/ClientHandoutsModal/ClientHandouts";
+import { SearchHit, transformHits } from "../../../models/SearchHits";
 import { icon } from "../../../assets";
-import { parseAlgoliaSchedule } from "../../../models";
 import styles from "./SearchResults.module.scss";
 
-/**
- * Transform Algolia search hits such that each hit has a recurringSchedule that
- * uses the time helper classes.
- */
-const transformHits = (hits) =>
-  hits.map((hit) => {
-    const inheritedSchedule =
-      hit.schedule && hit.schedule.length
-        ? hit.schedule
-        : hit.resource_schedule;
-    const recurringSchedule =
-      inheritedSchedule && inheritedSchedule.length
-        ? parseAlgoliaSchedule(inheritedSchedule)
-        : null;
-    return { ...hit, recurringSchedule };
-  });
-
-// Todo: setExpandList will be used as part of next stage of multiple location work
-// eslint-disable-next-line no-unused-vars
-const SearchResults = ({ searchResults, expandList, setExpandList }) => {
-  const [centerCoords, setCenterCoords] = useState(null);
-  const [googleMapObject, setMapObject] = useState(null);
+const SearchResults = ({ searchResults, expandList }) => {
+  const [centerCoords] = useState(null);
+  interface MapObjectProps extends GoogleMapProps {
```

Comment:
Is this type correct? The [google-map-react docs](https://github.com/google-map-react/google-map-react#use-google-maps-api) seem to indicate that the `onGoogleApiLoaded` prop gives you a map object, not a map props object. The map objects appears to be the actual object defined by the official Google Maps JavaScript API and not the google-map-react wrapper.

It looks like the [`@types/google.maps`](https://www.npmjs.com/package/@types/google.maps) NPM package contains type definitions that are automatically generated from the official Google Maps JS API, and it defines a [`Map`](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/f33d088a06faa38729454ae4278624bd84441b35/types/google.maps/index.d.ts#L3171) class that has a [`setCenter()`](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/f33d088a06faa38729454ae4278624bd84441b35/types/google.maps/index.d.ts#L3352) method on it, which I think is the actual type of the object you have here.

If this sounds right, then could you add `@types/google.maps` as a dependency and use that `Map` type instead of this `MapObjectProps` interface that you defined?

Reply:
Ah thanks! That was puzzling me. The `Map` class you suggested is correct

TIL -- interestingly the google.maps package has no exports but its types are added to the TS index